### ISSUE TITLE
Discuss adding social networking features

### ISSUE THREAD

**Issue by maeutic** [Syna3C/Syna3C-Project#31]
Revencher has suggested including social networking features in the beta, specifically friends and user dashboards.

**Comment by maeutic** [comment]
Personally, I don't see Syna3C as inherently needing a friends feature though user dashboards could be handy. If we include user to user "connections", I think it would make more sense to go with Twitter-style followers, so users could follow an organization. Better yet would be a subscription model, where users could subscribe/unsubscribe to events along with coordination threads and posts. 

Users following users in general doesn't seem to make much sense given the stated purposes and mission of the platform. If users end up liking one another through interactions online or in person, they can exchange contact info via the messaging system (whether implemented in the alpha or in a later release) or in person. I think including a friends function muddies the scope and mission, and could actually undermine the efficacy of the platform by encouraging off-topic discussion.

A dashboard makes a lot more sense: a central location where users can see activity of events they're involved with, responses to coordination threads/posts, new messages (once the messaging system is implemented), moderation queue if they're trusted contributors, events they've organized and summary of activity if they're organizers, etc. 

However, to make the dashboard really useful, a great deal more functionality will need to be included which is beyond the scope of the alpha, I think. Each event, coordination thread and post would need summary information related to quantity of resources needed/filled/unfulfilled by type, which would then be filtered and displayed to users differently depending on their roles and events/threads/posts they've engaged with. The check-in feature would also need to be implemented to allow organizers to easily see that information, with time-sensitive info prioritized to the top (e.g., distribution center is on fire or volunteers are missing in a flooded area with dangerous currents). *Aside: we probably need some kind of notification or flag to alert organizers of this kind of situation as well (persistent icon or modal on the platform when an organizer is logged in? push notifications? SMS feature?).*

We'll see where we are once the alpha is complete, but I suspect the dashboard feature would be beyond what is feasible with the beta and should be postponed for later release to ensure the beta primarily serves to fix bugs and other identified issues/immediately needed improvements.
